src/v/model/metadata.h

```diff
@@ -468,7 +468,8 @@ enum class cloud_storage_backend {
     google_s3_compat = 1,
     azure = 2,
     minio = 3,
-    unknown = 4,
+    oracle = 4,
```

Comment:
I'd call this oracle_s3_compat similar to google_s3_compat. It communicates better what type of implementation it is and is future proof just in case we will in the remote remote future decide to support native oracle apis.